core/config/config_test.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"chain/database/pg/pgtest"
+	"chain/database/sinkdb"
+	"chain/database/sinkdb/sinkdbtest"
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"testing"
+)
+
+func TestDetectStaleConfig(t *testing.T) {
+	defer func() {
+		if r := recover(); r == nil {
+			t.Errorf("Expected panic detecting stale config")
+		}
+	}()
+
+	ctx := context.Background()
+	sdb := sinkdbtest.NewDB(t)
+	_, db := pgtest.NewDB(t, pgtest.SchemaPath)
+	c := newTestConfig(t)
+
+	// Write config to sinkdb
+	sdb.Exec(ctx,
+		sinkdb.IfNotExists("/core/config"),
```

Comment:
Nit: this line isn't really necessary since we know it's a fresh sinkdb, but nbd.